src/components/avatar/Avatar.js

```diff
@@ -55,7 +55,7 @@ class Avatar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { avatarUrl, className, name, id, size }: Props = this.props;
+        const { avatarUrl, className, name, id, size = '' }: Props = this.props;
```

Comment:
I included this change as it helps clean up a flow error that got through the flow check, didn't think we would need a separate ticket + new pr just for a one liner change